File: rnd_number/VB.NET/rnd.vb
'Use Visual Studio or mono-basic to compile
'Generates and prints a random number from 0 and 1000.
Imports System

Module Program
    Sub Main(args As String())
        Dim random As New Random
        Dim rndNr As Integer
        rndNr = random.Next(0, 1001)
        Console.WriteLine("Your random number is: {0}", rndNr)
    End Sub
End Module

File: rnd_number/FreePascal/rnd.pas
{ Compile using fpc rnd.pas and run with ./rnd }
{ Generates a random number between 1 and 1000 and prints it to the screen }

program randomizer;

var 

    rnd: integer;

begin

    randomize;
    rnd := random(1000) + 1;    
    writeln ('Here is your random number: ', rnd);

end.
